Learn_go/代码随想录/project03-TCP/main.go

```
@@ -0,0 +1,49 @@
+
+	// Read data
+	buf := make([]byte, 1024)
+	n, err := conn.Read(buf)
+	if err != nil {
+		return
+	}
+
+	fmt.Println("Received data", buf[:n])
+
+	// Write the same data back
+	conn.Write(buf[:n])
```

golangci-lint (1.64.8)

Error return value of `conn.Write` is not checked

(errcheck)